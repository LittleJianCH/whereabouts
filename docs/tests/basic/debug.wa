query (x, y) debug {
  unify x = true
  unify y = false
}
