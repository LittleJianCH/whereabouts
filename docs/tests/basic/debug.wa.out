- solution: []
  goals:
    - unify "?x" = true
    - unify "?y" = false

- solution:
    - "true"
  goals:
    - unify "?y" = false

- solution:
    - "false"
    - "true"
  goals: []

[ [true, false] ]
